tools/webhook-apicoverage/resourcetree/resourcetree.go

```diff
@@ -62,4 +62,9 @@ func (r *ResourceTree) createNode(field string, parent NodeInterface, t reflect.
 func (r *ResourceTree) BuildResourceTree(t reflect.Type) {
 	r.Root = r.createNode(r.ResourceName, nil, t)
```

Comment:
should we make buildChildNodes as part of createNode? looks like it will make rest of code much simpler.

Reply:
createNode is the tree method that has the common code used by all node types, where as buildChildNodes as node specific implementation as each node type would have different implementation.